pkg/utils/oci/helmregistry.go

```diff
@@ -25,7 +25,7 @@ import (
 const (
 	mediaTypeHelm = "application/vnd.cncf.helm.chart.content.v1.tar+gzip"
 
-	inKubernetesRegistry = "local.gardener.cloud:"
+	inKubernetesRegistry = "garden.local.gardener.cloud:5001"
```

Comment:
The new default registry is `registry.gardener.cloud`. Why using the old one here in the constant?